slurm-infra.yml

```diff
@@ -40,7 +40,7 @@
   gather_facts: false
   tasks:
     - name: Set up Ansible user
-      user: "{{ appliances_local_users_ansible_user }}"
+      user: "{{ (appliances_local_users_default | selectattr('user.name', 'eq', appliances_local_users_ansible_user_name))[0]['user'] }}"
```

Comment:
What changed here so that we need this now?

Reply:
See https://github.com/stackhpc/ansible-slurm-appliance/commit/9893c355073aceeb4c16adcec9529fcab2ed4a33 - commit message explains.